moveit_ros/benchmarks/examples/demo_panda_predefined_poses.launch

```diff
@@ -0,0 +1,31 @@
+<launch>
```

Comment:
seems this should live in the tutorials... no?

Reply:
Not sure about that. There is already a set of similar examples. It's also really convenient to be able to customize these from within the main package.

Reply:
ideally the tutorials would live in this repo (future work)

in other components we've been keeping stuff like this in the tutorials, but your call.

we could move all of these examples into the tutorials... this package predates the moveit_tutorials repo